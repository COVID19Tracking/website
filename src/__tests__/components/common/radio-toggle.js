import React from 'react'
import renderer from 'react-test-renderer'
import RadioToggle from '~components/common/radio-toggle'

describe('Components : Common: Card', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <RadioToggle
          options={['Last 90 days', 'Full range']}
          state
          setState={() => {}}
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
