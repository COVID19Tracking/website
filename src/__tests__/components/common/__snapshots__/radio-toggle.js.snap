// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Common: Card renders correctly 1`] = `
<div
  className="toggle"
  role="radiogroup"
>
  <button
    aria-checked={false}
    className={false}
    onClick={[Function]}
    role="radio"
    title="Last 90 days"
    type="button"
  >
    Last 90 days
  </button>
  <button
    aria-checked={true}
    className="active"
    onClick={[Function]}
    role="radio"
    title="Full range"
    type="button"
  >
    Full range
  </button>
</div>
`;
